src/reynir_correct/errtokenizer.py

```diff
@@ -2118,9 +2159,11 @@ def is_valid_suggestion(
         # !!! TODO: We are not handling those here.
         # !!! TODO: Handle upper/lowercase
         if not token.val and CIDErrorForms.contains(token.txt):
-            corr_txt = CIDErrorForms.get_correct_form(token.txt)
-            rtok = replace_word(2, token, corr_txt, corr_txt)
-            at_sentence_start = False
+            rtok = token
+            if "S002" not in ignore_rules:
+                corr_txt = CIDErrorForms.get_correct_form(token.txt)
+                rtok = replace_word(2, token, corr_txt, corr_txt)
+                at_sentence_start = False
```

Comment:
Is the resetting of at_sentence_start correctly inside the if?

Reply:
Fixed